jsco4180Sup/detect_jasco_error_state.st

```diff
@@ -73,30 +89,26 @@ ss detect_jasco_error_state
                     if (status == 1) {
                         errlogSevPrintf(errlogMinor, "Resetting pump");
                         PVPUT(pump_reset, 1);
+                        epicsThreadSleep(1.0); /* or use SYNC on above? */
                         errlogSevPrintf(errlogMinor, "Restarting run");
```

Comment:
These four lines are shared for both cases of the if, can we move them out? Can you either change it to be sync as suggested or remove the comment.

Reply:
That was a question for somebody who knows the Jasco well - a SYNC would guarantee the order of execution, but I didn't know if a delay is also required after a reset or else it might ignore the next command. The previous code had no guarantee of anything, so I though at the very least a SYNC was likely required